homework3/task01/tests/test_cach_decorator.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+from project1.task1 import cache
+
+
+@cache(times=2)
+def f(a: int):
+    return a + 1
+
+
+def test_cache_decorator():
+    assert f(6) is f() is f()
```

Comment:
It is expected that you should cache the function result for each argument separately. 

```python
@cache(times=1)
def foo(arg):
   print(arg)
   return arg
```
In that case, if you call

foo(1)
foo(1)
foo(2)
foo(1)

You will get 3 lines in the console
1 (first call with 1)
2 (first call with 2)
1 (third call with 1)